### PR TITLE
Added Citations for Google Mobility Data

### PR DESCRIPTION
**Pull request by nss10**
Jira Ticket: [COV-1156](https://occ-data.atlassian.net/browse/COV-1156)

### Improvements
* Updated Citations for google mobility data

### CODE REVIEW

File: src/Covid19Dashboard/ControlPanel/index.jsx [hunk 1]
```diff
@@ -11,9 +11,10 @@ import './ControlPanel.less';
 
 class ControlPanel extends PureComponent {
   render() {
-    const tooltipText1 = 'Google Mobility Data shows movement trends as a percentage variance from an established baseline for a given day and place.';
+    const tooltipText1 = (<React.Fragment> Google Mobility Data shows movement trends as a percentage variance from an established baseline for a given day and place. <br /> Google LLC &quot;Google COVID-19 Community Mobility Reports&quot;. <a href='https://www.google.com/covid19/mobility/'> https://www.google.com/covid19/mobility/</a>  <br /> Accessed: Today</React.Fragment>);
```
Comment (paulineribeyre) [inline src/Covid19Dashboard/ControlPanel/index.jsx:14]: i think we might need to remove the spaces before and after the `<br />` tags?

**Review by paulineribeyre — COMMENTED** [review]
could we deploy this to QA so we can see it before merging?

File: src/Covid19Dashboard/ControlPanel/index.jsx (hunk 1 above)
Comment (nss10) [reply src/Covid19Dashboard/ControlPanel/index.jsx:14]: I don't think the spaces matter. The image in the slack thread looks right to me. ([here](https://slack-files.com/T03A08KRA-F03KLGZBW5D-8256f2ee86)) Let me know if you still want me to double check.

**Comment by nss10** [comment]
> could we deploy this to QA so we can see it before merging?

Absolutely. Deploying it in qa-covid19, will let you know when it is ready.

**Review by paulineribeyre — APPROVED** [review]

**Comment by nss10** [comment]
Ignored a few unrelated failed [tests](https://jenkins.planx-pla.net/blue/organizations/jenkins/CDIS_GitHub_Org%2Fdata-portal/detail/PR-1020/2/tests) since they were related to `etl` and `StudyViewer`

**Pull request merged**